ci/ray_ci/automation/determine_microcheck_tests.py

```diff
@@ -143,10 +143,8 @@ def _get_most_impact_test(
 
 def _get_failed_prs(test: Test, test_history_length: int) -> Set[str]:
     """
-    Get the failed PRs for a test. Currently we use the branch name as an identifier
-    for a PR.
-
-    TODO (can): Use the PR number instead of the branch name
+    Get the failed PRs for a test. We use the commit to account for all revisions
+    of a PR.
```

Comment:
this is not returning set of "PRs" any more? should we change it to `_get_failed_runs` or `_get_failed_commits`?